src/pytroll_watchers/publisher.py

```diff
@@ -98,22 +100,25 @@ def unpack_dir(path):
                     **path.storage_options)
 
 
-def _build_file_location(file_item, include_dir=None):
+def _build_file_location(file_item, include_dir=None, no_fs=False):
     file_location = dict()
-    file_location["uri"] = as_uri(file_item)
-    if include_dir:
-        uid = include_dir + file_item.path.rsplit(include_dir, 1)[-1]
-    else:
-        uid = file_item.name
-    file_location["uid"] = uid
     with suppress(AttributeError):  # fileitem is not a UPath if it cannot access .fs
         with dummy_connect(file_item):
             file_location["filesystem"] = json.loads(file_item.fs.to_json(include_password=False))
+        if no_fs:
+            raise ValueError("Can’t have no_fs with remote files…")
```

Comment:
```suggestion
            raise ValueError("Can't have no_fs with remote files…")
```

I think we usually prefer ASCII source code (’ is RIGHT SINGLE QUOTATION MARK, 0x2019).

Reply:
Ah indeed, sorry. I had forgotten how limited the ascii char set was…
Now, although I agree on the change, for me it’s more a matter of consistency with all other strings we have, since we already output unicode in a few place, right? the nbsp use in the wavelength unit output pops to mind…

Reply:
Oh, maybe requesting ASCII applies to identifiers and not to strings in source code.  I don't remember for sure now.  It's not important to me, I just noticed the ‘ that taken a compose key combination for me to type :)